ledger/participant-state/kvutils/tools/README.md

```diff
@@ -34,53 +34,28 @@ The following options are supported:
 
 # `benchmark-replay`
```

Comment:
Let's just maybe rename this one for clarity.

Reply:
Indeed !